### PR TITLE
added isValidPhone & isValidCreditCard utility

### PR DESCRIPTION
**Pull request by vipul-dixit-2004**
I have currently added the isValidPhone and isValidCreditCard utility. Working on Date Format Validator, will commit it soon.

### CODE REVIEW

File: src/utils/utilityFunctions.ts [hunk 1]
```diff
@@ -77,3 +77,36 @@ export const isURL = (str: string) => {
   // test the string against the regular expression
   return urlRegex.test(str);
 };
+
+export const isValidPhone=(num:string):boolean=>{
```
Comment (Jagroop2001) [inline src/utils/utilityFunctions.ts:81]: this is not valid way of checking card is valid or not.Please take reference of this blog : 
https://learnersbucket.com/examples/javascript/credit-card-validation-in-javascript/#:~:text=Validating%20credit%20card%20in%20javascript%20with%20Luhn's%20algorithm.&text=Get%20a%20double%20of%20every,divisible%20then%20it%20is%20valid.

**Comment by jagroop-dev-11** [comment]
@vipul-dixit-2004 , please add these in readme.md file as well

**Comment by vipul-dixit-2004** [comment]
okay @Jagroop2001 on it!!

**Comment by vipul-dixit-2004** [comment]
Thankyou @Jagroop2001 to point this out, I have edited the Readme.md file.

**Comment by jagroop-dev-11** [comment]
@vipul-dixit-2004 , please resolve this PR conflict.

**Comment by vipul-dixit-2004** [comment]
Sorry Sir, can you please guide me with this I am not able to understand what to do.

**Comment by jagroop-dev-11** [comment]
> Sorry Sir, can you please guide me with this I am not able to understand what to do.

I have managed this . Don't worry

**Comment by jagroop-dev-11** [comment]
https://github.com/Jagroop2001/validify-string/issues/5

**Comment by vipul-dixit-2004** [comment]
Thank you Sir

**Pull request merged**